src/test_application/TestSystemInterface.h

```diff
@@ -0,0 +1,5 @@
+#include "../../3rdparty/RmlUi/Include/RmlUi/Core.h"    //TODO: Still cannot include RmlUi in new files
+
+class TestSystemInterface : Rml::SystemInterface {
+
+};
```

Comment:
There should always be a (blank) newline at the end of text files. Some programs may not consider this a valid line and ignore it.

POSIX standards (practically anything non-windows) say that every line should be terminated by a newline character.

Unix text editors will add the blank line automatically, but apparently in VS there's a hidden setting: https://stackoverflow.com/questions/52894653/visual-studio-2017-insert-final-newline

Reply:
Noted, thanks for the feedback